js/pin.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+window.pin = (function () {
+  var pinElementTemplate = document.querySelector('#pin').content.querySelector('.map__pin');
+
+  return {
+    generatePins: function () {
+      var pinDocFrag = document.createDocumentFragment();
+      for (var i = 0; i < window.data.nearbyLisitings.length; i++) {
```

Comment:
can we move this loop to the map, map has the data and then just call`window.pin.generate` locally. Just one pin,

Reply:
Done

Reply:
When you say done Push the code up. I don't see any changes in the code.